test/resources.e2e-spec.ts

```diff
@@ -0,0 +1,320 @@
+import { Test, TestingModule } from "@nestjs/testing";
+import { INestApplication, ValidationPipe } from "@nestjs/common";
+import { seed } from "../prisma/seed/seed";
+import * as request from "supertest";
+import { AppModule } from "../src/app.module";
+import { PrismaService } from "../src/prisma/prisma.service";
+import { CreateResourceDto } from "src/resources/dto/create-resource.dto";
+import { UpdateResourceDto } from "src/resources/dto/update-resource.dto";
+import { extractResCookieValueByKey } from "./utils";
+
+const loginUser = async (
+    email: string,
+    password: string,
+    app: INestApplication,
+) => {
+    const res = await request(app.getHttpServer())
+        .post("/auth/login")
+        .send({
+            email,
+            password,
+        })
+        .expect(200);
+
+    return extractResCookieValueByKey(
+        res.headers["set-cookie"],
+        "access_token",
+    );
+};
+
+const findVoyageTeamId = async (email: string, prisma: PrismaService) => {
+    return await prisma.voyageTeamMember.findFirst({
+        where: {
+            member: {
+                email: email,
+            },
+        },
+        select: {
+            userId: true,
+            voyageTeamId: true,
+        },
+    });
+};
+
+describe("ResourcesController (e2e)", () => {
```

Comment:
could you please categories the tests by endpoints, you can look at auth, techs e2e tests for example